dio/lib/src/form_data.dart

```diff
@@ -30,20 +30,23 @@ class FormData {
   /// Whether [finalize] has been called.
   bool get isFinalized => _isFinalized;
   bool _isFinalized = false;
+  final bool _camelCaseContentDisposition;
 
-  FormData() {
+  FormData({bool camelCaseContentDisposition = false}) {
+    _camelCaseContentDisposition = camelCaseContentDisposition;
     _init();
   }
 
   /// Create FormData instance with a Map.
-  FormData.fromMap(
-    Map<String, dynamic> map, [
+  FormData.fromMap(Map<String, dynamic> map, [
     ListFormat collectionFormat = ListFormat.multi,
+  {bool camelCaseContentDisposition = false}
```

Comment:
This is impossible when positioned optional arguments are set.

Reply:
you are right, my mistake